template/bigbot/src/rest/mod.ts

```diff
@@ -42,7 +42,13 @@ async function handleRequest(conn: Deno.Conn) {
       );
     }
 
-    const json = (await requestEvent.request.json());
+    // deno-lint-ignore no-explicit-any
+    let json: any;
```

Comment:
Why no ternary like
const json = requestEvent.request.body ? await ..... : undefined;

Reply:
You're right, that would be much nicer!